pkgs/top-level/python-packages.nix

```diff
@@ -6009,7 +6009,7 @@ in {
 
   sure = callPackage ../development/python-modules/sure { };
 
-  svgwrite = callPackage ../development/python-modules/svgwrite { };
+  svgwrite = disabledIf (pythonAtLeast "3.8") (callPackage ../development/python-modules/svgwrite { });
```

Comment:
this should be moved into `svgwrite/default.nix `.

see `disabled` parameter https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters

Reply:
Okay, but why are many other packages in nixpkgs using ``disabledIf`` in the same way that I did?